utils/find-overlay-dependencies.sh

```diff
@@ -68,16 +68,22 @@ for sdk in ${(k)SDKS}; do
   DEPENDS_ON=()
   printf "%s:\n\t" "$sdk"
   deps=$(echo "@import $1;" | xcrun -sdk $sdk clang -arch $arch -x objective-c - -M -fmodules 2>/dev/null)
+  if [[ $? != 0 ]]; then
+    # Don't `continue` here because we want to clear out the SDK line if it exists
```

Comment:
Warnings get lost in the console spew. I'd rather make it an error and stop the script or remove the line entirely and let the programmer catch it in the ``git diff``. The danger is if there is a supported platform and the ``clang`` line somehow errors.